ServerCore/Pages/Puzzles/SubmitFeedback.cshtml

```diff
@@ -26,7 +26,7 @@
             </div>
             <div class="form-group">
                 <label asp-for="Feedback.Difficulty" class="control-label"></label>
-                <input asp-for="Feedback.Difficulty" type="range" min="0" max="10" value="5" step="1" class="slider" id="difficultySlider">
+                <input asp-for="Feedback.Difficulty" type="range" min="0" max="10" step="1" class="slider" id="difficultySlider">
```

Comment:
Nit - should these be constants somewhere so we can enforce them on the model side as well?

Reply:
Adding min/max as constants and clamping them before adding the feedback to the DB.